### PR TITLE
Revert "Add secret and CLI option for CSA emails to Docker compositions"

### PR DESCRIPTION
**Pull request by dav3r**
#  #

## 🗣 Description ##

This PR reverts commit 458d2b2a06373cea29905e187419b875738bf4f7, which (once deployed) will cause cyhy-mailer to stop BCCing the CSAs on weekly CyHy reports and daily CyHy notifications.

## 💭 Motivation and context ##

This was requested in https://github.com/cisagov/cyhy-system/issues/129, please refer to that issue for more information.

## 🧪 Testing ##

All current automated tests pass.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]
I think you should bump the version and create a release, though, since this particular repo contains both the Python code and the Docker configuration.

File: cyhy/mailer/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 """This package contains the cyhy-mailer code."""
 
-__version__ = "1.7.4"
```
Comment (mcdonnnj) [inline cyhy/mailer/__init__.py]: Does this require a version bump? The Docker composition files aren't attached to the logic of the Docker image or the Python package which should be what dictates version changes.

**Review by mcdonnnj — APPROVED** [review]
Just one nitpick but otherwise LGTM.

File: cyhy/mailer/__init__.py (hunk 1 above)
Comment (jsf9k) [reply cyhy/mailer/__init__.py]: I'm fine if you want to remove the version bump.

Comment (dav3r) [reply cyhy/mailer/__init__.py]: I was on the fence, but I went with Shane's suggestion to bump the version.  I don't think it makes much difference either way.

Comment (dav3r) [reply cyhy/mailer/__init__.py]: @mcdonnnj Can we resolve this conversation?

Comment (mcdonnnj) [reply cyhy/mailer/__init__.py]: :/ I'm not a fan of bumping versions for no functionality change. If the two of you want to bump it then fine, but those Docker composition files really belong in [cisagov/ansible-role-cyhy-mailer](https://github.com/cisagov/ansible-role-cyhy-mailer) to mirror updates to other, similar Docker setups such as [cisagov/ansible-role-code-gov-update](https://github.com/cisagov/ansible-role-code-gov-update) and [cisagov/ansible-role-vdp-scanner](https://github.com/cisagov/ansible-role-vdp-scanner).

Comment (dav3r) [reply cyhy/mailer/__init__.py]: I think we are saying the same thing.  Until those Docker composition files move, they are essentially part of the functionality of this repo.  Totally agree that those files should be moved elsewhere, but for now, in the interest of not spending any more time on this, I'd like to just move ahead as is.

Comment (mcdonnnj) [reply cyhy/mailer/__init__.py]: My point is the version tracks the Python package, and as a result the Docker image. The Docker composition files just happen to live here and If you download the [Docker image](https://hub.docker.com/r/cisagov/cyhy-mailer) or install this Python package directly these files are nowhere to be found.

Comment (jsf9k) [reply cyhy/mailer/__init__.py]: I'm fine if you want to remove the version bump.

Comment (dav3r) [reply cyhy/mailer/__init__.py]: The version bump has been removed.

**Review by jsf9k — APPROVED** [review]

**Comment by dav3r** [comment]
FYI, I manually applied these changes to the Production `reporter` and `docker` instances.

**Pull request merged**